src/contentscript/overlay/root/components/Dapplet/index.tsx

```diff
@@ -24,7 +24,7 @@ export interface DappletProps
   index?: any
   onSwitchChange: Function
   onSettingsModule: Function
-  onOpenDappletAction: Function
+  onOpenDappletAction: any
   onRemoveMyDapplet?: Function
   onDeployDapplet: Function
   onOpenStore: Function
```

Comment:
The `any` and `Function` types are useless and will not help you write a type-safe code. 
If you'd rewrite the callbacks like `onOpenDappletAction: any` => `onOpenDappletAction: (dapplet: SomeDappletType) => void` you can use the power 🔥 of TypeScript in this line: `onClick={() => onOpenDappletAction(dapplet)}`